scripts/scenarios/cloud6/cloud6-4nodes-compute-ha.yml

```diff
@@ -0,0 +1,199 @@
+---
+proposals:
+- barclamp: pacemaker
+  name: services
+  attributes:
+    stonith:
+      mode: libvirt
+      libvirt:
+        hypervisor_ip: ""
```

Comment:
Does this make sense to leave empty with libvirt mode? Or are you going to change it to SBD, just like we have in QA jobs?